standards.py

```diff
@@ -7,6 +7,7 @@
 import datetime
 import subprocess
 
+assert sys.version_info >= (3, 7)
```

Comment:
if we could update this to `3, 9` at least, I think we could keep the change on line 1